lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -1340,7 +1340,14 @@ void ByteCodeGenerator::DefineUserVars(FuncInfo *funcInfo)
 
             if (sym->GetSymbolType() == STVariable)
             {
-                if (fGlobal)
+                if (sym->GetIsModuleExportStorage())
```

Comment:
Is it necessary to do this initialization of export memory at runtime? It seems like it could be done when we allocate the memory.